kythe/release/appengine/buildbot/cloudbuild.yaml

```diff
@@ -0,0 +1,29 @@
+steps:
+- name: gcr.io/cloud-builders/gsutil
+  id: 'Fetch secrets'
+  args:
+  - cp
+  - gs://kythe-buildbot/secrets.tar.enc
+  - secrets.tar.enc
+- name: gcr.io/cloud-builders/gcloud
+  id: 'Decrypt secrets'
+  args:
+  - kms
+  - decrypt
+  - --location=global
+  - --keyring=Buildbot
+  - --key=secrets
+  - --ciphertext-file=secrets.tar.enc
+  - --plaintext-file=secrets.tar
+- name: gcr.io/cloud-builders/docker
+  id: 'Build image'
+  args:
+  - build
+  - -t
+  - gcr.io/$PROJECT_ID/buildbot.$_VERSION
+  - .
+timeout: 1800s # 30 minute timeout
```

Comment:
actually I'm fairly certain you can just write 30m

This just gets parsed as a duration or whatever

Reply:
The documentation suggests otherwise (as do all of the examples):
```
timeout
Use the timeout field in a build step to set a time limit for executing the step. If you don't set this field, the step has no time limit and will be allowed to run until either it completes or the build itself times out. timeout must be specified in seconds with up to nine fractional digits, terminated by 's'. Example: "3.5s"
```

Reply:
Ah I was just looking around at random examples:

https://github.com/GoogleCloudPlatform/golang-samples/blob/master/testing/cloudbuild/cloudbuild.yaml#L15

the underlying proto is a Duration

I would at least try it once with 30m and see if it breaks

Reply:
ERROR: (gcloud.builds.submit) INVALID_ARGUMENT: Invalid value at 'build.timeout' (type.googleapis.com/google.protobuf.Duration), Field 'timeout', Illegal duration format; duration must end with 's'
- '@type': type.googleapis.com/google.rpc.BadRequest
  fieldViolations:
  - description: Invalid value at 'build.timeout' (type.googleapis.com/google.protobuf.Duration),
      Field 'timeout', Illegal duration format; duration must end with 's'
    field: build.timeout